packages/server/graphql/types/Team.ts

```diff
@@ -115,11 +115,11 @@ const Team = new GraphQLObjectType<ITeam, GQLContext>({
       type: GraphQLISO8601Type,
       description: 'The datetime the team was last updated'
     },
-    customPhaseItems: {
-      type: new GraphQLList(CustomPhaseItem),
+    reflectPrompts: {
+      type: new GraphQLList(ReflectPrompt),
       resolve: ({id: teamId}, _args, {dataLoader}) => {
         // not useful for retros since there is no templateId filter
-        return dataLoader.get('customPhaseItemsByTeamId').load(teamId)
+        return dataLoader.get('reflectPromptsByTemplateId').load(teamId)
```

Comment:
templateId is not available here, but i don't think this field needs to exist anymore. can add `deprecatedReason` to it & return an empty array for now

Reply:
what's deprecatedReason?

Reply:
graphql lets you optionally add a `deprecationReason` to fields that are no longer used. https://graphql.org/graphql-js/type/